### PR TITLE
Add star on required fields

### PR DESCRIPTION
**Pull request by pkoch**

### CODE REVIEW

**Comment by andresgnlez** [comment]
image

@pkoch I think setting a category of an impact is mandatory as well, could you confirm this, please?

File: src/components/form/Form.jsx [hunk 1]
```diff
@@ -6,9 +6,13 @@ function withWrapper(Component) {
   return class extends React.Component { // eslint-disable-line react/prefer-stateless-function
     render() {
       const { label, ...props } = this.props; // eslint-disable-line react/prop-types
+
+      const required = (props.validations || []).find(e => e == 'required');
```
Comment (andresgnlez) [inline src/components/form/Form.jsx]: Could you use `===` instead of `==`, please? I'm sure eslint will be happier like this :)

Comment (pkoch) [inline src/components/form/Form.jsx]: You're absolutely right. It got lost in the middle of the other eslint complaints. :/ Can we ship #35 and add a precommit hook so this doesn't happen again?

Comment (andresgnlez) [inline src/components/form/Form.jsx]: Sure! I didn't want to merge anything in order to not interrupt your workflow, but if the PR is approved feel free to merge it whenever you need.

Comment (pkoch) [inline src/components/form/Form.jsx]: :white_check_mark:

**Comment by pkoch** [comment]
I'll get back to this once we have stabilized the whole form. Will make sure to address those categories.

**Pull request merged**